workshops/chat/README.md

```diff
@@ -301,7 +320,7 @@ Congratulations. Your chat app is complete. Save all your files and commit your
 - `git commit -m "Create a chat app using Firebase"`
 - `git push`
 
-**After entering your username and password, you'll need to add `USERNAME.github.io` to the list of Authorized Domains in Firebase.**
+**After entering your username and password, you will need to add `USERNAME.github.io` to the list of Authorized Domains in Firebase.** You can see how we did this earlier in [Configuring Permissions](#configuring-permissions).
```

Comment:
What if we do this in the [Configuring Permissions](#configuring-permissions) step so hackers don't have to go back later?